icu4c/source/i18n/unicode/plurrule.h

```diff
@@ -42,10 +42,18 @@
  */
 #define UPLRULES_NO_UNIQUE_VALUE ((double)-0.00123456777)
 
+/**
+ * A FixedDecimal version of UPLRULES_NO_UNIQUE_VALUE used in PluralRulesTest
+ * for parsing of samples.
+ * @stable ICU 68.1
```

Comment:
We can't make this "born stable". We only use "born stable" for things like constants that correspond to stable Unicode property values.

It also can't really be public because of FixedDecimal.

Please move it to plurrule_impl.h and remove the API status tag.

Reply:
Please do move it to plurrule_impl.h. There should be no need for this in the public header file. The test code can and does include internal headers.

Reply:
Done.